ostd/Cargo.toml

```diff
@@ -58,7 +58,12 @@ iced-x86 = { version = "1.21.0", default-features = false, features = [
     "gas",
 ], optional = true }
 
+[target.riscv64gc-unknown-none-elf.dependencies]
+riscv = { version = "0.11.1", features = ["s-mode"] }
+sbi-rt = "0.0.3"
+fdt = { version = "0.1.5", features = ["pretty-printing"] }
+
 [features]
-default = ["intel_tdx", "log_color"]
```

Comment:
It seems hard, because the `default` features are applied to all targets. In other words, if `intel_tdx` is included in `default` features, other non-x86 targets would also have `intel_tdx` enabled. This is weird and doesn't make sense.

I prefer something like this but it's unfortunately unsupported yet, see rust-lang/cargo#1197.
``` toml
[target.'cfg(target_arch = "x86_64")'.features]
default = ["intel_tdx", "log_color"]

[target.'cfg(target_arch = "riscv64")'.features]
default = ["log_color"]
```

We may need to enable `intel_tdx` feature manually (i.e. `--features intel_tdx`) in TDX CI.

Reply:
I have discussed with the TDX developer @Hsy-Intel , and he will submit a PR to place all code under `#[cfg(feature = "intel_tdx")]` to under `#[cfg(all(target_arch = "x86_64",feature = "intel_tdx" )]`, so it will be possible to set `intel_tdx` as default feature after that change?

Reply:
Technically, it can. But from my point of view, semantically, enabling `intel_tdx` feature on non-x86 architecture doesn't make any sense, although the actual code is not enabled.

Reply:
You are correct, `intel_tdx` is not compatible with riscv. Therefore, it would be best for the feature to be independent of architecture. As a result, #1111 has renamed the feature as `cvm_guest` (confidential virtual machine). Since riscv can also support its cvm, this feature may be suitable for all targets. Hopefully, this name change will help you feel comfortable in keeping it as the default feature.